harness/determined/cli/model.py

```diff
@@ -34,8 +34,7 @@ def render_model(model: Model) -> None:
     render.tabulate_or_csv(headers, [values], False)
 
 
-def render_model_version(model_version: ModelVersion) -> None:
-    checkpoint = model_version.checkpoint
+def render_model_versions(model_versions: List[ModelVersion]) -> None:
```

Comment:
i don't see why `render_model_version` needs to be a public method. it should just be for internal use. so i changed this method to `_render_model_versions`. we should still track this as it's _technically_ a breaking change though i don't think it would affect anyone. i've added a section to the SDK confluence to track breaking changes.

Reply:
Awesome x2. Thanks!